index.html

```diff
@@ -2630,8 +2630,8 @@ <h2>Definition of Roles</h2>
 		<div class="role" id="complementary">
 			<rdef>complementary</rdef>
 			<div class="role-description">
-				<p>A <rref>landmark</rref> that is designed to be complementary to the main content at a similar level in the DOM hierarchy, but remaining meaningful when separated from the main content.</p>
-				<p>There are various types of content that would appropriately have this <a>role</a>. For example, in the case of a portal, this may include but not be limited to show times, current weather, related articles, or stocks to watch. The complementary role indicates that contained content is relevant to the main content. If the complementary content is completely separable from the main content, it may be appropriate to use a more general role.</p>
+				<p>A <rref>landmark</rref> that is designed to be complementary to the main content it is a direct descendant of, or is at a similar level in the DOM hierarchy. The contents of a complementary landmark would be expected to remain meaningful if it were to be separated from the main content it is relevant to.</p>
```

Comment:
Not sure about this. I think what we mean is that it should be either a sibling landmark or a child landmark in the landmark hierarchy. I'm not sure why it matters where it is in the DOM hierarchy. (ie <main><div><div><div><complementary></div></div></div>...</main> should be absolutely fine). I don't think we've ever defined a concept of landmark hierarchy though so don't think we can use this terminology.

Reply:
simplifying the terminology sounds good to me.  I think "DOM hierarchy" is likely a cause of confusion for people here anyway, so I probably should have gotten rid of that term anyway.